rwgame/debugstate.cpp

```diff
@@ -235,27 +237,10 @@ void DebugState::tick(float dt)
 		}
 	}*/
 
-	if( _freeLook ) {
```

Comment:
Add this check back please.
The debug menu is spinning around even if freelook is not selected.